### PR TITLE
feat(forms): Add forms package

### PR DESCRIPTION
**Pull request by eckdanny-osi**
### [CL-83](https://osi-cwds.atlassian.net/browse/CL-83)

- [x] `libs/forms` exists
- [x] jest works
- [x] babel works
- [x] dev (babel)
- [x] build (tsc)
- [x] build (babel)
- [x] build (rollup)
- [x] using in `www`/`rails_app` works

### CODE REVIEW

**Comment by eckdanny-osi** [comment]
👍 Let's hang on this until after we cut `1.0.0.` ([JIRA release](https://osi-cwds.atlassian.net/projects/CL/versions/10987/tab/release-report-all-issues))

**Comment by eckdanny-osi** [comment]
> 👍 Let's hang on this until after we cut `1.0.0.` ([JIRA release](https://osi-cwds.atlassian.net/projects/CL/versions/10987/tab/release-report-all-issues))

shipped today. Picking this up again.

File: libs/forms/src/SelectList.module.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.SelectList {
+  border: 5px solid red;
```
Comment (mlipardo) [inline libs/forms/src/SelectList.module.scss]: Not sure what this style for to have red border?

Comment (eckdanny-osi) [reply libs/forms/src/SelectList.module.scss]: probably curried in from long ago. Guessing this was just to verify the [s]css module loading worked

Comment (mlipardo) [reply libs/forms/src/SelectList.module.scss]: 👍

**Pull request merged**